recipe/meta.yaml

```diff
@@ -13,8 +13,8 @@ source:
   - folder: {{ pkg_src }}
     git_url: https://github.com/helm/helm  # [unix]
     git_tag: v{{ version }}                # [unix]
-    url: https://storage.googleapis.com/kubernetes-{{ name }}/{{ name }}-v{{ version }}-windows-amd64.tar.gz  # [win]
-    sha256: 604780d3fabeb27e7ab7a30c6e29ce64bcd2203501ea35e5231c97965b0255a0  # [win]
+    url: https://get.helm.sh/{{ name }}-v{{ version }}-windows-amd64.zip  # [win]
```

Comment:
Switched to the url in the GitHub release page as 3.0.0 isn't yet available from `storage.googleapis.com`


https://storage.googleapis.com/kubernetes-helm